atlasdb-cassandra/src/main/java/com/palantir/atlasdb/cassandra/CassandraServersConfigs.java

```diff
@@ -131,30 +131,25 @@ public CqlCapableConfigTuning tuning() {
         }
 
         @Override
-        public final int numberOfHosts() {
-            return hosts().size();
-        }
-
-        @Value.Derived
-        public Set<InetSocketAddress> thriftHosts() {
-            return constructHosts(thriftPort());
+        public final int numberOfThriftHosts() {
+            return thriftHosts().size();
         }
 
-        @Value.Derived
-        public Set<InetSocketAddress> cqlHosts() {
-            return constructHosts(cqlPort());
-        }
+        @Value.Check
+        final void check() {
+            checkPortNumbers(thriftHosts(), "'thriftPort'");
 
-        private Set<InetSocketAddress> constructHosts(int port) {
-            return hosts().stream()
-                    .map(host -> new InetSocketAddress(host, port))
-                    .collect(Collectors.toSet());
+            checkPortNumbers(cqlHosts(), "'cqlPort'");
```

Comment:
In general for these, it looks fine - but do we need the single quotes around the names?

Reply:
Kept it because it was like that previously.